.github/workflows/ci.yml

```diff
@@ -1,7 +1,7 @@
 name: Build macOS, Ubuntu, Windows
 
 on:
-  push:
+  pull_request:
```

Comment:
tags are not not valid for the `pull_requst` trigger, see documentation: 

this should just be (similar to the `workflow_dispatch` trigger):

```yaml
on:
  pull_request:
```

Reply:
Thank you for documentation link. Fixed.